### PR TITLE
Product Link Functionality: Select Products Pagination

### PR DESCRIPTION
**Pull request by ilyaskarim**

### CODE REVIEW

File: src/fullPageBookingForm/Components/Common/CustomForm/ProductAddons.tsx [hunk 1]
```diff
@@ -36,94 +42,140 @@ export const ProductAddons: FunctionComponent<ProductAddonsProps> = (
...
-          }
-
-          if (isSelectedProduct) {
-            classNames.push("active");
-          }
-
           return (
             <li
-              className={classNames.join(" ")}
+              className={classnames({
```
Comment (ckeyes88) [inline src/fullPageBookingForm/Components/Common/CustomForm/ProductAddons.tsx:124]: Nice abstraction of the classnames.

File: src/fullPageBookingForm/Components/Common/QuantitySelection/QuantitySelection.stories.tsx [hunk 2]
```diff
@@ -13,23 +14,24 @@ const defaultOnIncreaseClick = () => {};
 const defaultOnDecreaseClick = () => {};
 
 const defaultArgs: QuantitySelectionProps = {
+  labels: defineLanguageDictionary("en-US"),
```
Comment (ckeyes88) [inline src/fullPageBookingForm/Components/Common/QuantitySelection/QuantitySelection.stories.tsx:17]: is this hard coded to english?

File: src/fullPageBookingForm/Components/TimeslotGroup/TimeslotGroup.stories.tsx [hunk 3]
```diff
@@ -9,45 +9,32 @@ export default {
...
-    />
-  </div>
-);
+export const Basic = () => {
+  const labels = defineLanguageDictionary("en-US")
+  return (
+    <div style={{ width: 460 }}>
+      <TimeslotGroup
+        labels={labels}
+        languageCode={"en-US"}
```
Comment (ckeyes88) [inline src/fullPageBookingForm/Components/TimeslotGroup/TimeslotGroup.stories.tsx:18]: We shouldn't be hard coding the language.

File: src/fullPageBookingForm/Components/TimeslotGroup/TimeslotGroup.stories.tsx [hunk 4]
```diff
@@ -9,45 +9,32 @@ export default {
...
+              new Date("March 17, 2021 10:00:00"),
+              "America/Los_Angeles",
+            ),
+            remainingSpots: 4,
+            minPrice: 150,
+            timezone: "America/Los_Angeles",
+            onSelect: () => {},
+            moneyFormat: "${{amount}}",
+            labels: labels,
+            languageCode: "en-US",
```
Comment (ckeyes88) [inline src/fullPageBookingForm/Components/TimeslotGroup/TimeslotGroup.stories.tsx:34]: Same here.

File: src/fullPageBookingForm/Components/Views/Confirmation/Confirmation.stories.tsx [hunk 5]
```diff
@@ -10,6 +11,7 @@ export default {
 
 const defaultArgs: ConfirmationProps = {
   email: "test@test.com",
+  labels: defineLanguageDictionary("en-US"),
```
Comment (ckeyes88) [inline src/fullPageBookingForm/Components/Views/Confirmation/Confirmation.stories.tsx:14]: Hard coded language

**Review by ckeyes88 — CHANGES_REQUESTED** [review]
Just a couple questions that may need changes.

File: src/fullPageBookingForm/Components/Common/CustomForm/CustomForm.tsx [hunk 6]
```diff
@@ -206,6 +206,11 @@ export const CustomForm: FunctionComponent<CustomFormProps> = ({
              * to each variant name by slicing into the field array with each
              * variant name.
              */}
+
+            <div>
+              <TextStyle text={"Attendee Info"} variant="body2"/>
```
[the remark this answers is not in the document]
Comment (ilyaskarim) [reply src/fullPageBookingForm/Components/Common/CustomForm/CustomForm.tsx:211]: Oh Yes I will fix this.

**Comment by ilyaskarim** [comment]
@ckeyes88 the hard coded language is for development purposes only. I will add an option to change language from stories books playground.

File: example/index.html [hunk 7]
```diff
@@ -7,14 +7,17 @@
     <title>Event Gallery Widget Dev</title>
     <!-- <link href="/dist/calendarBookingForm.css" rel="stylesheet" /> -->
```
[the remark this answers is not in the document]
Comment (ilyaskarim) [reply example/index.html:8]: @ckeyes88 Alright.

**Comment by ckeyes88** [comment]
@ilyaskarim sorry about the hard coded comments, I didn't catch that those were in story files. I think for now it's fine to leave as english and we can add other language support later.

**Comment by ckeyes88** [comment]
@ilyaskarim just let me know when this is ready to re-review so I know when pull it in.

**Comment by ilyaskarim** [comment]
Hey chip, yesterday I was working on GitHub action. Today I'll execute this
and let you know.

**Review by ckeyes88 — APPROVED** [review]

**Pull request merged**